src/app/contextBroker/orionRestServices.cpp

```diff
@@ -506,9 +507,8 @@ static RestService badVerbV[] =
 
 /* ****************************************************************************
 *
-* optionsV - options service vector to be used if CORS is enabled
+* optionsV - 
 *
-* Adds API_V2_CORS definitions on top of the default service vector (restServiceV)
 */
```

Comment:
Not sure why the comment about "options service vector to be used if CORS is enabled" was removed.
It's valid, right?

Reply:
Please see my comment below. NTC